### ISSUE TITLE
Nullable decimals Ignore Scale and Precision

### ISSUE THREAD

**Issue by daniele-olivieri-rs** [Sielnix/EFCore.Snowflake#9]
When generating a DbMigration model-first, nullable decimals ignore the `precision` and `scale` in the `type` string.

For example:
```
public class Entity
{
    public decimal? PurchasePrice { get; set; }
}

public class EntityConfiguration  :  IEntityTypeConfiguration<Entity>
{
    public void Configure(EntityTypeBuilder<Entity> builder)
    {
      builder.Property(x => x.PurchasePrice).HasPrecision(18, 2);
    }
}
```

Produces this in the DbMigration:
```
PURCHASE_PRICE = table.Column<decimal>(type: "NUMBER(38,18)", precision: 18, scale: 2, nullable: true),
```

Notice, `type` is `"NUMBER(38,18)"` instead of `"NUMBER(18,2)"`

**Comment by Sielnix** [comment]
Hello @daniele-olivieri-rs ,

Thank you for your bug report. This issue has been addressed and fixed in [v8.0.4.1](https://www.nuget.org/packages/EFCore.Snowflake/8.0.4.1). Please update the package and verify if it works.

Please be advised, that if you are using migrations, then after upgrade next added migration will change column type from `NUMBER(38,18)` to `NUMBER(18,2)` in your case.

**Comment by daniele-olivieri-rs** [comment]
Thanks @Sielnix!
